store/migration/dev/LATEST.sql

```diff
@@ -659,7 +659,8 @@ CREATE TABLE task (
     status TEXT NOT NULL CHECK (status IN ('PENDING', 'PENDING_APPROVAL', 'RUNNING', 'DONE', 'FAILED', 'CANCELED')),
     type TEXT NOT NULL CHECK (type LIKE 'bb.task.%'),
     payload JSONB NOT NULL DEFAULT '{}',
-    earliest_allowed_ts BIGINT NOT NULL DEFAULT 0
+    earliest_allowed_ts BIGINT NOT NULL DEFAULT 0,
+    rollback_by INTEGER NOT NULL DEFAULT 0
 );
```

Comment:
Also why it's a weak reference instead of enforcing foreign key constraint

Reply:
Because many tasks do not have a rollback_by task.

Reply:
Seems like foreign key columns can be NULL. Changed to use foreign key constraint.